pkg/controller/account/account_controller.go

```diff
@@ -660,9 +660,41 @@ func (r *ReconcileAccount) initializeRegions(reqLogger logr.Logger, currentAcctI
 		return err
 	}
 
+	if !currentAcctInstance.IsBYOC() { // This is a normal account creation
+		reqLogger.Info("Normal account region initialization")
+		go r.asyncRegionInit(reqLogger, currentAcctInstance, creds, regionAMIs, castAWSRegionType(regionsEnabledInAccount.Regions))
+		return nil
+	}
```

Comment:
@ArielLima its possible at the moment for a non BYOC user to select this region which should result in a failed cluster install. I think we can remove this so that "normal" AWS account claims fail.

Reply:
The problem with this is if we remove it for non-ccs accounts, they're created before the AccountClaim because they're created by the accountpool.  So there would be no "wantedRegion" in those accounts because there's no accountclaim.

Instead we could check in the accountclaim controller to fail if it's a non-ccs account AND it's selecting a non-standard region.

Reply:
@jharrington22 In addition to what @iamkirkbater just said, I thought the  idea was that “normal” accounts wouldn’t have the option of being one of the 4 opt-in regions so it wouldn’t be an issue. Additionally while that isn’t yet implemented this will still work. Reminder, **it only failed because we set it up to fail** - it won’t fail again since the region was re-enabled.

@iamkirkbater I would much rather improve the comment than check in the accountclaim controller since it is much simpler the way it currently is. As you like to say though, I won’t die on this hill but it would take more time to implement.

Reply:
Yeah, that could definitely be another body of work.  It would be better to just remove the option from being able to be selected by OCM, but I know they have a different deployment schedule than we do.  I'll create a card here to do the check and we can circle back to that.